runtime/testy/Ansible.py

```diff
@@ -222,16 +228,101 @@ def start(self):
 
         try:
             while True:
-                nextCall = time.time()
                 event = sel.select()
```

Comment:
Could we use the sleeps for throttling reasons as we will be probably be receiving files faster than we want to be running this thread?